build.gradle

```
@@ -21,14 +21,20 @@ repositories {
 dependencies {
     implementation 'org.springframework:spring-context:4.0.1.RELEASE'
     implementation 'org.springframework:spring-web:5.1.6.RELEASE'
-
+    
 
     implementation 'javax.validation:validation-api:1.0.0.GA'
     implementation 'commons-lang:commons-lang:2.6'
     implementation 'net.lingala.zip4j:zip4j:1.2.4'
     implementation 'com.fasterxml.jackson.core:jackson-annotations:2.9.8'
     implementation 'commons-codec:commons-codec:1.10'
 
+    compile group: 'org.aspectj', name: 'aspectjrt', version: '1.9.4'
```

```suggestion
    implementation 'org.aspectj:aspectjrt:1.9.4'
```

Seit gradle 3.0 ist compile deprecated. Hier kannst du auch die Kurzform verwenden